doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -188,8 +183,9 @@ wb.download_project_archive( archive_name="my_project_archive")
 
 ## PyMechanical
```

Comment:
```suggestion
## Link PyMechanical
```

Reply:
Link and operate title will more appropriate. Thanks!